.github/workflows/run.yml

```diff
@@ -0,0 +1,46 @@
+name: Run
+
+on:
+  workflow_dispatch:
+
+jobs:
+  packager:
+    name: Packager
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: 
+        type:
+          - full
```

Comment:
Just making sure I understand, but if we wanted to publish `roots/wordpress` with the no-content version, then we'd just change this type here right?

Reply:
Exactly